deploy/kubernetes/csi-nodeplugin-rbac.yaml

```diff
@@ -1,13 +1,14 @@
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: csi-nodeplugin
+  name: cvmfs-nodeplugin
+  namespace: cvmfs
```

Comment:
As above.

Reply:
The namespace is not strictly needed.
But the name should contain cvmfs in the it, to avoid conflicts.